src/shared/utilities/messages.ts

```diff
@@ -0,0 +1,20 @@
+/*!
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+
+import * as nls from 'vscode-nls'
+const localize = nls.loadMessageBundle()
+
+export function makeCheckLogsMessage(): string {
+    const commandName = localize('AWS.command.viewLogs', 'View AWS Toolkit Logs')
+    const message = localize(
+        'AWS.error.check.logs',
+        'Check the logs for more information by running the {0} command from the Command Palette.',
```

Comment:
Nit: For clarity, suggest to wrap `{0}` in quotes:

>     Check the logs for more information by running the "View AWS Toolkit Logs" command from the Command Palette.

Reply:
Done